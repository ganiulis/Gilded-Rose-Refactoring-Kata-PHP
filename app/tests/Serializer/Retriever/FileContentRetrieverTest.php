<?php

namespace App\Tests\Serializer\Retriever;

use App\Serializer\Retriever\FileContentRetriever;
use ApprovalTests\Approvals;
use PHPUnit\Framework\TestCase;

class FileContentRetrieverTest extends TestCase
{
    public function setUp(): void
    {
        $this->retriever = new FileContentRetriever();
    }

    public function testRetrieveTxt(): void
    {
        $content = $this->retriever->retrieve(__DIR__.'/testContent.txt');

        Approvals::verifyString($content);
    }

    public function testRetrieveCsv(): void
    {
        $content = $this->retriever->retrieve(__DIR__.'/testContent.csv');

        Approvals::verifyString($content);
    }

    public function testRetrieveJson(): void
    {
        $content = $this->retriever->retrieve(__DIR__.'/testContent.json');

        Approvals::verifyString($content);
    }
}
