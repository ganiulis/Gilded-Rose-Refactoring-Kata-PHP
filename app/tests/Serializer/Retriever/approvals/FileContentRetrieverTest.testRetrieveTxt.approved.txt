Hello, world! This is test content for FileContentRetrieverTest.php.
As you can see, I was really hungry when I made this particular test.
However, it works, so it's OK.
